src/js/utils.js

```diff
@@ -1,66 +1,69 @@
 export const getFromLSOrDefaultValue = (key, defaultValue) => {
   const item = localStorage.getItem(key);
-
   return item ? JSON.parse(item) : defaultValue;
-}
+};
 
 export const createSearchFilter = (formData) => {
-  const {
-    user,
-    repository,
-    typeOfSearch,
-    condition,
-    date
-  } = formData;
+  const { user, repository, typeOfSearch, condition, date } = formData;
 
-  let searchCondition = '';
+  let searchCondition = "";
```

Comment:
Can you add a rule to EsLint to change double quotes for single quotes in javascript code?